integrationtest/jdk/java-modules/pom.xml

```diff
@@ -17,6 +17,8 @@
     <description>Hibernate Search integration tests for (JDK 11+) Java modules</description>
 
     <properties>
+        <test.elasticsearch.run.skip>false</test.elasticsearch.run.skip>
```

Comment:
Shouldn't there be a `<test.database.run.skip>false</test.database.run.skip>` too? Just for correctness.